File: Linguagem C#/Vetores/SomaVetor.cs
using System;
using System.Globalization;

namespace SomaVetor {
    class Program {
        static void Main(string[] args) {

            CultureInfo CI = CultureInfo.InvariantCulture;

            int n;
            double soma, media;

            Console.Write("Quantos numeros voce vai digitar? ");
            n = int.Parse(Console.ReadLine());

            double[] vet = new double[n];

            for (int i = 0; i < n; i++) {
                Console.Write("Digite um numero: ");
                vet[i] = double.Parse(Console.ReadLine(), CI);
            };

            Console.Write("VALORES = ");
            for (int i = 0; i < n; i++) {
                Console.Write(vet[i].ToString("F1",CI)+"  ");
            };
            Console.WriteLine("");

            soma = 0;
            for (int i = 0; i < n; i++) {
                soma = soma + vet[i];
            };
            media = soma / n;
            Console.WriteLine("SOMA = " + soma.ToString("F2",CI));
            Console.WriteLine("MEDIA = "+media.ToString("F2",CI));
        }
    }
}

File: Linguagem Python/Vetores/SomaVetor.py
n = int(input("Quantos numeros voce vai digitar? "))

vet= [0 for x in range(n)]

for i in range (0,n):
    vet[i] = float(input("Digite um numero: "))

print()
print("VALORES = ",end="")
for i in range(0,n):
    print(f"{vet[i]:.1f} ",end="")

print()
soma = 0;
for i in range (0,n):
    soma = soma + vet[i]

media = soma / n
print(f"SOMA = {soma:.2f}")
print(f"MEDIA = {media:.2f}")
